admin/tests/unit/controllers/authenticated/certification-centers/get-test.js

```diff
@@ -0,0 +1,211 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import Service from '@ember/service';
+import sinon from 'sinon';
+
+module('Unit | Controller | authenticated/certification-centers/get', function(hooks) {
+
+  setupTest(hooks);
+
+  let certificationCenter;
+  let createRecordStub;
+  let saveStub;
+
+  let controller;
+
+  hooks.beforeEach(function() {
+    sinon.restore();
+
+    controller = this.owner.lookup('controller:authenticated/certification-centers/get');
+
+    createRecordStub = sinon.stub();
+    saveStub = sinon.stub();
+
+    saveStub.resolves();
+    createRecordStub.returns({
+      save: saveStub,
+    });
+
+    certificationCenter = { id: 1 };
+
+    controller.store = Service.create({
+      createRecord: createRecordStub,
+    });
+
+    controller.model = {
+      certificationCenter,
+    };
+
+    controller.notifications = {
+      success: sinon.stub(),
+      error: sinon.stub(),
+    };
+    controller.notifications.success.resolves();
+    controller.notifications.error.resolves();
+  });
+
+  module('#updateEmailErrorMessage', function() {
+
+    test('should set error message if email syntax is invalid', function(assert) {
+      // given
+      controller.userEmailToAdd = 'an invalid email';
+
+      // when
+      controller.send('updateEmailErrorMessage');
+
+      // then
+      assert.equal(controller.errorMessage, controller.EMAIL_INVALID_ERROR_MESSAGE);
+    });
+
+    test('should set email error message to null if email is empty', function(assert) {
+      // given
+      controller.errorMessage = 'error message';
+      controller.userEmailToAdd = '';
+
+      // when
+      controller.send('updateEmailErrorMessage');
+
+      // then
+      assert.equal(controller.errorMessage, null);
+    });
+  });
+
+  module('#addCertificationCenterMembership', function(hooks) {
+
+    let event;
+
+    hooks.beforeEach(function() {
+      event = { preventDefault() {} };
+      controller.send = sinon.stub();
+    });
+
+    module('when email is valid', function() {
+
```

Comment:
**chipotage**
Extraire dans le module et ensuite référencer dans les it `controller.userEmailToAdd = email;`
```suggestion
const validEmail = 'test@example.net';
```

Reply:
Done